scripts/build_scripts/build_script_osx_ssl.sh

```diff
@@ -57,10 +65,6 @@ cp scripts/launchers/unplatform_osx_ssl.sh bundle/
 mkdir bundle/modules
 cp -r modules/* bundle/modules/
 
-# copy over utility files for FSP data extraction
-cp scripts/data_extraction/DataExtractionScript.bat bundle/
```

Comment:
Since they're ``.bat`` files and unable to run on OSX, I thought that they served no purpose.  I'm open to other opinions.  :-)